vector/src/gplay/java/im/vector/app/nightly/NightlyProxy.kt

```diff
@@ -44,4 +53,24 @@ class NightlyProxy @Inject constructor() {
                     }
                 }
     }
+
+    private fun canDisplayPopup(): Boolean {
+        if (BuildConfig.APPLICATION_ID != "im.vector.app.nightly") return false
```

Comment:
do we need the application id guard if the logic lives within the nightly sourceset?

Reply:
It's in the `gplay` sourceset. I cannot add to `nightly` sourceset because of `initWith release`. So I cannot have a version for `release`, one for `debug` and one for `nightly` as I would like to, else I got duplicated class declaration .

Normally this popup would not affect GplayRelease version, but better safe than sorry.

Reply:
ah! thanks for explaining :+1: